tests/unit/runner/test_api.py

```diff
@@ -7,27 +7,36 @@
 
 
 @pytest.mark.parametrize(
-    "orig_exec, orig_args, sh_c_arg",
+    "orig_exec, orig_args, exp_stdout, exp_interactive",
```

Comment:
While you're at it, could you list separate args separately, plz?
```suggestion
    ("orig_exec", "orig_args", "exp_stdout", "exp_interactive"),
```